src/main/views/macro/support.njk

```diff
@@ -44,9 +44,23 @@
 {% endmacro %}
 
 {% macro _holidayWidget() %}
-  {% if (toDate() < toDate("2021-01-02") and toDate() >= toDate("2020-12-01")) %}
-    <h2 class="heading-medium contact-us"> {{ t('Christmas hours')}}</h2>
-    <p>{{ t("We're closed from 24 to 28 December and 1 January.")}}</p>
+  {% if (toDate() < toDate("2022-04-19") and toDate() >= toDate("2022-04-08")) %}
+    <h2 class="heading-medium contact-us"> {{ t('Easter opening times')}}</h2>
+    <p>{{ t("14 April - 8:00am - 5:00pm")}}</p>
+    <p>{{ t("15 April Closed")}}</p>
+    <p>{{ t("16 April Closed")}}</p>
+    <p>{{ t("17 April Closed")}}</p>
+    <p>{{ t("18 April Closed")}}</p>
+    <p>{{ t("19 April - Normal Times apply")}}</p>
+  {% endif %}
+  {% if (toDate() < toDate("2022-06-06") and toDate() >= toDate("2022-05-26")) %}
+    <h2 class="heading-medium contact-us"> {{ t('Platinum Jubilee opening times')}}</h2>
```

Comment:
to date should be less than or equal to 6th june. As the dates should be visible on 6th june also